erpnext/patches/v16_0/set_posting_datetime_for_sabb_and_drop_indexes.py

```
@@ -0,0 +1,31 @@
+def drop_indexes():
+	table = "tabSerial and Batch Bundle"
+	index_list = ["voucher_no_index", "item_code_index", "warehouse_index", "company_index"]
+
+	for index in index_list:
+		if not frappe.db.has_index(table, index):
+			continue
+
+		try:
+			frappe.db.sql_ddl(f"ALTER TABLE `{table}` DROP INDEX `{index}`")
+			click.echo(f"✓ dropped {index} index from {table}")
+		except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)